src/auth/flows.rs

```diff
@@ -53,7 +55,9 @@ pub fn config(cfg: &mut ServiceConfig) {
             .service(set_email)
             .service(verify_email)
             .service(subscribe_newsletter)
-            .service(link_trolley),
+            .service(link_trolley)
+            // auth/oauth
+            .configure(oauth::config),
```

Comment:
it's weird that this is under auth IMO, cld we just make it top level?

Reply:
yup